infra/notifications/src/emailSend.js

```diff
@@ -19,10 +18,140 @@ if (!process.env.SENDGRID_FROM_EMAIL) {
   logger.warn('Warning: SENDGRID_FROM_EMAIL env var is not set')
 }
 
+const {
+  getMessageFingerprint,
+  isNotificationDupe,
+  logNotificationSent
+} = require('./dupeTools')
+
+//
+// Email notifications for Messages
+//
+async function messageEmailSend(receivers, sender, config) {
+  if (!receivers) throw 'receivers not defined'
```

Comment:
nit: I believe in the back-end codebase we've been trying to throw new Error() rather than just string

Reply:
Yes, done!